### PR TITLE
Add SSFangTangTi for better Chinese characters rendering

### PR DESCRIPTION
**Pull request by xkww3n**
The original font YurukaStd has poor Chinese characters support. I added a Chinese font *Shangshou Fangtang Ti* (*上首方糖体* in Chinese, *SSFangTangTi* in font family name) to achieve better Chinese characters rendering.
The official description of this font can be found here:<http://www.ssfonts.com/shows/9/43.html>

### CODE REVIEW

**Comment by TheOriginalAyaka** [comment]
I don't know much about Chinese, i tested random characters and found that YurukaStd is working fine, I tested the font you added and there's some characters that doesn't work, if you are sure that this font is better for Chinese, let me know and I'll merge it with small regex check to detect when the input is in Chinese.
image

**Comment by xkww3n** [comment]
> I don't know much about Chinese, i tested random characters and found that YurukaStd is working fine, I tested the font you added and there's some characters that doesn't work, if you are sure that this font is better for Chinese, let me know and I'll merge it with small regex check to detect when the input is in Chinese. image

First, the SSFangTangTi is definitely better able to display Chinese characters.
But adding check for Chinese characters isn't a good idea. There are some info that I hope you could understand, otherwise this may break the existing good experience for Japanese users.
TLDR: There's no way to distinguish between Chinese, Japanese and Korean accurately.
**C**hinese, **J**apanese and **K**orean characters, as well as **CJK** characters, actually have shared some characters, and that's why some Chinese characters display fine in Japanese font.
But there are still many differences between Chinese and Japanese characters. For example, a common Chinese character `嘻` (laugh) doesn't exist in Japanese, so this character will fallback to default font if only defined a Japanese font, YurukaStd for example.
The Chinese character in the above image is an ancient Chinese character. Modern Chinese fonts don't have these ancient characters, and there's no need to consider them.
All in all, I suggest you'd better not to distinguish CJK characters, but just let our clever browsers to handle this situation.

**Comment by TheOriginalAyaka** [comment]
I guess I understand now, thanks for the explanation.
